public/css/index.css

```diff
@@ -140,21 +277,58 @@
 }
 
 .faq-section {
-	background-color: rgb(242, 204, 181);
+	background-color: rgb(255, 225, 149);
 	padding: 20px 75px 0px 75px;
 }
 
-.questions h2 {
-	color: rgb(42, 45, 78);
-	font-size: 20px;
-	font-weight: bold;
-	display: inline-block;
+.q-a {
+	background-color: rgba(255, 255, 255, 0.3);
+	padding: 15px 15px 10px;
+	margin-bottom: 25px;
 }
 
-.questions p {
-	color: rgb(64, 105, 143);
-	text-align: justify;
-	font-size: 15px;
+.question-section {
+	background-color: rgba(255, 255, 255, 0.5);
+	border-left: 10px solid #ec3fa6;
+	padding-left: 15px;
+}
+
+.answer-section {
+	padding-top: 10px;
+	padding-left: 25px;
+}
+
+.q,
+.a {
+	float: left;
+	width: 40px;
+
+	font-size: 40px;
+	font-weight: 700;
+	margin: 0;
+}
+
+.a {
+	color: #ec3fa6;
+}
+
+.question,
+.answer {
+	width: calc(100% - 40px);
+
+	float: left;
+	padding-left: 15px;
+}
+
+.question {
+	margin: 12px 0 0;
+	font-size: 22px;
+	font-weight: 700;
+}
+
+.answer {
+	margin: 17px 0 0;
+	font-size: 16px;
```

Comment:
This one is on me, but when a question and/or answer wrap, they end up a little too close to the bottom of the box they're in. I think you should be able to fix this by changing the bottom margin for both to be 10px

Reply:
Fixed!